src/engraving/rw/compat/compatutils.cpp

```diff
@@ -560,3 +564,56 @@ void CompatUtils::resetStemLengthsForTwoNoteTrems(MasterScore* masterScore)
         }
     }
 }
+
+void CompatUtils::replaceStaffTextWithCapo(MasterScore* score)
+{
+    TRACEFUNC;
+
+    std::set<StaffTextBase*> oldCapoSet;
+
+    for (Measure* measure = score->firstMeasure(); measure; measure = measure->nextMeasure()) {
+        for (Segment* segment = measure->first(); segment; segment = segment->next()) {
+            for (EngravingItem* annotation : segment->annotations()) {
+                if (!annotation || !annotation->isStaffTextBase()) {
+                    continue;
+                }
+
+                StaffTextBase* text = toStaffTextBase(annotation);
+
+                if (text->capo() > 1) {
+                    oldCapoSet.insert(text);
+                }
```

Comment:
I think you need `else { continue; }` here. Otherwise, linked elements are added even if not `text->capo() > 1`.

Reply:
oops :)